content/python/concepts/dunder-methods/terms/init/init.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: '__init__()'
+Description: 'Initializes a newly created object and is called each time a new class instance is created.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Attributes'
+  - 'Classes'
+  - 'Methods'
+  - 'Objects'
+  - 'OOP'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`__init__()`** method initializes a newly created object. It is called each time a new instance of the defined class is created.
+
+## Syntax
+
+```pseudo
+class ClassName:
+  __init__(self, param1, param2, ..., paramN):
+    self.param1 = param1
+    self.param2 = param2
+    self.paramN = paramN
+```
+
+Instance variables (`param1, param2, ..., paramN`) are set within the `__init__()` method block. They can be set initially within the `__init__()` method when the instance is created, or instance variables can be added by setting them to values:
+
+```pseudo
+new_instance = ClassName()
```

Comment:
Can we add a line here with the instance variable being set to some value?

Reply:
I made a change in 5e1ebc1b5d0ee952ef5d7bd65e4f45d24e77c9c9 per your feedback. What do we think?

Reply:
Looks good!